src/config/index.ts

```diff
@@ -81,13 +81,15 @@ export const getNetworkConfigDisabledFeatures = (): SafeFeatures => getConfig()?
 
 export const getNetworkInfo = (): NetworkSettings => getConfig()?.network
 
-export const getTxServiceUriFrom = (safeAddress: string) => `/safes/${safeAddress}/transactions/`
+export const getSafeServiceUri = (safeAddress: string) => `safes/${safeAddress}`
 
-export const getIncomingTxServiceUriTo = (safeAddress: string) => `/safes/${safeAddress}/incoming-transfers/`
+export const getTxServiceUriFrom = (safeAddress: string) => `${getSafeServiceUri(safeAddress)}/transactions`
 
-export const getAllTransactionsUriFrom = (safeAddress: string) => `/safes/${safeAddress}/all-transactions/`
+export const getIncomingTxServiceUriTo = (safeAddress: string) => `${getSafeServiceUri(safeAddress)}/incoming-transfers`
```

Comment:
I'm starting to feel some concerns about this approach. I see that config is getting a bit more complex when you want to search how to query a param and we already have utility functions to build the URLs. Won't it make sense to just have the base route in the config and the specialized routes in the buildUrl files?

Reply:
I agree with the complexity. And makes sense to have only the base route in configs. But I don't feel comfortable with the buildUrl files distributed across the whole app.

WDYT about having a different file that that holds all the buildUrl function, closest to the configs?

Reply:
done